### ISSUE TITLE
SM: Can not convert object to number

### ISSUE THREAD

**Issue by karaxuna** [jxcore/jxcore#519]
Throws exception when running [code](https://github.com/karaxuna/jxcore-tutorial-ghost-packaged) with `jx index` command, but works with `node index`.

![bug](https://cloud.githubusercontent.com/assets/2152815/9699002/9499d1fc-53e1-11e5-938c-ad33f500ac63.png)

**Comment by obastemur** [comment]
@karaxuna give it a shot with jx(V8) ?

**Comment by karaxuna** [comment]
@obastemur yes, it worked with v8

**Comment by ktrzeciaknubisa** [comment]
Although the issue for SM seems to be already fixed lately with 78e1a151b6e39810ced435518bbe8abbfa37a69a, I see another thing with SM, too:

```
ERROR: SQLITE_CONSTRAINT:NOT NULL constraint failed: settings.created_at 
 The app will not be loaded 
 Check with the app creator, or read the app documentation for more details on app requirements 
```

Despite of the error message, the app starts, however soon enough it crashed for me:

```bash
 NotFoundError@/[...]/jxcore-tutorial-ghost-packaged/node_modules/ghost/core/server/errors/not-found-error.js:6:18
```

**Comment by ktrzeciaknubisa** [comment]
I moved SM related error to a separate issue #540.
Closing this one.
